### ISSUE TITLE
iOS compilation error in OneSignal Unity SDK 2.3.2

### ISSUE THREAD

**Issue by boffman** [OneSignal/OneSignal-iOS-SDK#247]
I just downloaded the 2.3.2 unitypackage, for unity 5, and got these errors for the iOS platform:

```
OneSignal/src/OneSignal.cs(341,123): error CS0103: The name 'logLevel' does not exist in the current context

Assets/OneSignal/src/OneSignal.cs(341,133): error CS0103: The name 'visualLogLevel' does not exist in the current context

Assets/OneSignal/src/OneSignal.cs(341,148): error CS1502: The best overloaded method match for 'OneSignalIOS.OneSignalIOS(string, string, bool, bool, OneSignal.OSInFocusDisplayOption, OneSignal.LOG_LEVEL, OneSignal.LOG_LEVEL)' has some invalid arguments

Assets/OneSignal/src/OneSignal.cs(341,148): error CS1503: Argument '#6' cannot convert 'object' expression to type 'OneSignal.LOG_LEVEL'
```

(..the Android platform built without problem though! 👍 Our batch build actually started working again with this upgrade)

**Comment by omar92** [comment]
same problem for me

**Comment by jkasten2** [comment]
@boffman @omar92 This was fixed in the Unity 2.3.3 SDK update.
https://github.com/OneSignal/OneSignal-Unity-SDK/releases
